### PR TITLE
[@tools-website] Add mainnet to network select

### PR DESCRIPTION
**Pull request by KristinaSpasevska**

### CODE REVIEW

File: packages/apps/tools/src/components/Global/AddNetworkModal/AddNetworkModal.tsx [hunk 1]
```diff
@@ -120,7 +120,11 @@ export const AddNetworkModal: FC<IAddNetworkModalProps> = (props) => {
                   </Stack>
                 </section>
                 <section className={formButtonStyle}>
-                  <Button type="submit" icon="TrailingIcon">
+                  <Button
+                    type="submit"
+                    icon="TrailingIcon"
+                    disabled={Boolean(error)}
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/components/Global/AddNetworkModal/AddNetworkModal.tsx:126]: Out of curiosity; why was this done / why is this needed? 😁

Comment (KristinaSpasevska) [reply packages/apps/tools/src/components/Global/AddNetworkModal/AddNetworkModal.tsx:126]: To prevent submitting if there are errors, such as duplicate networks etc

Comment (sanderlooijenga) [reply packages/apps/tools/src/components/Global/AddNetworkModal/AddNetworkModal.tsx:126]: Thanks for the explanation 👍

**Review by sanderlooijenga — APPROVED** [review]
